auto_instrument_handler/index.js

```diff
@@ -0,0 +1,26 @@
+const lumigo = require('@lumigo/tracer')({});
+
+const ORIGINAL_HANDLER_KEY = 'LUMIGO_ORIGINAL_HANDLER';
+
+const handler = (event, context, callback) => {
+  const originalHandler = process.env[ORIGINAL_HANDLER_KEY];
+  if (!originalHandler) {
```

Comment:
Also, check the format and throw a readable error to the user, that he does not get an error on the split and slice functions

Reply:
Great idea!